src/main/cpp/audioengine.cpp

```diff
@@ -327,7 +327,7 @@ namespace AudioEngine
                 for ( int k = 0; k < lAmount; ++k )
                 {
                     BaseAudioEvent* vo = channel->liveEvents[ k ];
-                    channelBuffer->mergeBuffers( vo->synthesize( amountOfSamples ), 0, 0, 1.0f );
+                    vo->mixBuffer( channelBuffer );
                 }
```

Comment:
Just render audio directly into the channels buffer, this omits the need of audio events allocating their own private buffer for temporary rendering.